products/chromeos.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Chrome OS
+category: os
+tags: linux-distribution
+iconSlug: googlechrome
+permalink: /chromeos
+alternate_urls:
+-   /chrome-os
+versionCommand: cat /etc/issue
+releasePolicyLink: https://chromeos.dev/en/education/chromeos-lts
+changelogTemplate: https://support.google.com/chrome/a/answer/10314655/__RELEASE_CYCLE__
```

Comment:
```suggestion
changelogTemplate: https://support.google.com/chrome/a/answer/10314655/__RELEASE_CYCLE__
changelogTemplate: https://support.google.com/chrome/a/answer/10314655#__RELEASE_CYCLE__
```

It's does not look like it is working with a '/'. Links in the `Chrome version & Stable channel release date` are using a dash.